symphony/bdk/core/config/model/bdk_agent_config.py

```diff
@@ -0,0 +1,7 @@
+from symphony.bdk.core.config.model.bdk_client_config import BdkClientConfig
```

Comment:
If I remember well, this class was used to accommodate for the load-balancing usecase. So you can remove it,

Reply:
I shouldn't have pushed this in the first place... 😄